app/flux/API/UserAPI.js

```diff
@@ -0,0 +1,34 @@
+var User = {
+    get: () => {
+        return fetch('/api/users')
```

Comment:
I find it odd that all the other stores follow a Module > method (get/put/post) -> operation pattern except this one.